python/cudf/cudf/_lib/json.pyx

```diff
@@ -42,6 +43,15 @@ from cudf._lib.types cimport dtype_to_data_type
 from cudf._lib.utils cimport data_from_unique_ptr, table_view_from_table
 
 
+def _get_json_recovery_mode(object on_bad_lines):
+    if on_bad_lines.lower() == "error":
```

Comment:
Would have a preference that `on_bad_lines` would not be normalized with `lower`, but not a blocker